### DISCUSSION TITLE
Convert to Bitmap node (Prerender/Bake)

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6565] [Ideas]
A node to transform the data to bitmap. This is because not all Fx response in the expected way in all the situations, so I suposse that a node like that could act like a bitmap to be able to apply all Fx you need without problem.

**Comment by RodneyBaker** [comment]
I'm renaming the title of this request in order to try to capture the intent clearly.
It is my understanding that the goal would be to have a node that allows a prerendering of the flow up to the point of that node that can be shared with nodes downstream.  This is basically a 'convert to bitmap' or 'convert to raster' or 'rasterize' process but likely one that renders to disk so that data can be baked in and therefore passed on downstream into node that will only accept input from columns/levels.

To expand on this idea even further (and of questionable sanity) I can see a set of negative frames hidden by default from the Xsheet/Timeline where this prerendering would be exposed.  In turn the Schematic nodes would references this hidden/negative frames as they woud be true columns/levels... just not ordinarily seen.  These exposures would form the potential that could be used if referenced.

Added:  It is not quite clear to me how this prerendering would fully account for keyframing events but my assumption would be that the rasterization would take into account all keyframes.
